testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -756,7 +779,9 @@ def test_positions(self):
         ag.positions = pos
         # should work
         assert_almost_equal(ag.coordinates(), pos,
-                            err_msg="failed to update atoms 12:42 position to new position")
+                            err_msg="failed to update atoms 12:42 position "
+                            "to new position")
```

Comment:
Even though it is not exactly what PEP 8 says, I would align the two pieces of the string for clarity. Having the whole string in between parenthesis would silence syntax checkers.